igvm_defs/src/lib.rs

```diff
@@ -331,6 +331,12 @@ pub enum IgvmVariableHeaderType {
     ///
     /// IGVM specific extensions can be found in the [`dt`] module.
     IGVM_VHT_DEVICE_TREE = 0x312,
+    /// A parameter which holds a u32 bitfield value defining environmental
+    /// state of the VM.  The bitfield is defined by the IgvmEnvironmentInfo
+    /// structure.  The loader will write the state to the specified offset of
+    /// the specified parameter area.  The parameter location information is
+    /// specified by a structure of type [`IGVM_VHS_PARAMETER`].
```

Comment:
Also, could we add a doc link to the corresponding structure the host will deposit?